morph_tool/nrnhines.py

```diff
@@ -314,7 +314,7 @@ def _to_be_isolated(morphology_path, point):
             return nrnhines.point_to_section_end(cell.icell.all, point)
 
         def _isolated(morph_data):
-            return nrnhines.isolate(_to_be_isolated)(*morph_data)
+            return nrnhines.isolate(_to_be_isolated)(\\*morph_data)
```

Comment:
Why the slashes here?

Reply:
Because without it sphinx thinks we want to emphasize text and complains that the emphasis block is not terminated.

Reply:
Does adding `>` help? ex:
```
>        def _to_be_isolated(morphology_path, point):
>            cell = nrnhines.get_NRN_cell(morphology_path)
>            return nrnhines.point_to_section_end(cell.icell.all, point)
>        def _isolated(morph_data):
>            return nrnhines.isolate(_to_be_isolated)(*morph_data)
```

Reply:
I ended up using `.. code-block::` which is more relevant for this.

Reply:
it looks a bit funny when reading a docstring w/ `help(isolate)`; but not sure anyone will do that.

Reply:
Ah yeah true. But I think it's explicit enough for people to understand what it means.